### PR TITLE
Add contao.org Top Navigation

### PR DESCRIPTION
**Pull request by fkaminski**
See: #1240
Adopted from contao.org.

### CODE REVIEW

File: page/layouts/partials/top-navigation.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<div id="contao-network">
+  <a href="https://contao.org" title="contao.org" class="contao active"><i></i> <span>Contao</span></a>
+  <a href="https://docs.contao.org" title="docs.contao.org" class="docs"><i></i> <span>Docs</span></a>
+  <a href="https://community.contao.org" title="community.contao.org" class="community"><i></i> <span>Community</span></a>
+  <a href="https://themes.contao.org" title="themes.contao.org" class="themes"><i></i> <span>Themes</span></a>
+  <a href="https://extensions.contao.org" title="extensions.contao.org" class="extensions"><i></i> <span>Extensions</span></a>
+  <a href="https://fanshop.contao.org" title="fanshop.contao.org" class="fanshop"><i></i> <span>Fanshop</span></a>
+  <a href="https://login.contao.org" title="login.contao.org" class="login"><i></i> <span>Login</span></a>
+</div>
```
Comment (fritzmg) [inline page/layouts/partials/top-navigation.html]: We integrate the network HTML everywhere via the https://contao.org/files/js/network.min.js

Comment (fkaminski) [reply page/layouts/partials/top-navigation.html]: Done.

File: page/static/css/theme-contao.css [hunk 2]
```diff
@@ -967,3 +967,90 @@ html[data-theme=dark] #toc-menu,
...
+
+#contao-network a.login i {
+  background: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIyMCIgaGVpZ2h0PSIyMCIgdmlld0JveD0iMCAwIDI0IDI0IiBmaWxsPSJub25lIiBzdHJva2U9IiNmZmYiIHN0cm9rZS1saW5lY2FwPSJyb3VuZCIgc3Ryb2tlLWxpbmVqb2luPSJyb3VuZCI+PHJlY3QgeD0iMyIgeT0iMTEiIHdpZHRoPSIxOCIgaGVpZ2h0PSIxMSIgcng9IjIiIHJ5PSIyIj48L3JlY3Q+PHBhdGggZD0iTTcgMTFWN2E1IDUgMCAwIDEgMTAgMHY0Ij48L3BhdGg+PC9zdmc+) 0 0 no-repeat
+}
+
+@media (max-width: 767px) {
+  #contao-network a span {
+      display:none;
+  }
+}
```
Comment (zoglo) [inline page/static/css/theme-contao.css]: Same here @fkaminski, there is a dedicated css file that we can load as well :)
https://contao.org/files/js/network.min.css

Please make sure to add it to the `<head>`:
```html
<link type="text/css" rel="stylesheet" href="https://contao.org/files/js/network.min.css">
```

Comment (fkaminski) [reply page/static/css/theme-contao.css]: Done

File: page/layouts/partials/custom-footer.html [hunk 3]
```diff
@@ -1,3 +1,4 @@
+<script src="https://contao.org/files/js/network.min.js"></script>
```
Comment (zoglo) [inline page/layouts/partials/custom-footer.html:1]: Just a nitpick, this could be in the `header.html` as well

Comment (fkaminski) [reply page/layouts/partials/custom-footer.html:1]: Yepp @fritzmg: As you like?

Comment (fritzmg) [reply page/layouts/partials/custom-footer.html:1]: On https://contao.org, https://docs.contao.org and https://partners.contao.org it is at the end of the `<body>`.

Comment (netzarbeiter) [reply page/layouts/partials/custom-footer.html:1]: Thank you, Franko. Will you include the script at the end of the `body` as Fritz suggested?

Comment (fkaminski) [reply page/layouts/partials/custom-footer.html:1]: @netzarbeiter It is already, next to the other scripts, before the closing body tag?

**Review by netzarbeiter — APPROVED** [review]

**Pull request merged**